### PR TITLE
PRT-152 update offset helper

### PR DESCRIPTION
**Pull request by lazaralex98**
I have updated the OffsetHelper to use redeemAuto2 and made a few other small changes in the meanwhile (check commit descriptions). I didn't manage to get retireAndMintCertificate to work for some reason, but I will open a follow-up issue to tackle this in the future.

### CODE REVIEW

File: contracts/OffsetHelper.sol [hunk 1]
```diff
@@ -257,60 +282,46 @@ contract OffsetHelper is OffsetHelperStorage {
...
-            address tco2 = scoredTCO2s[i];
-            uint256 balance = IERC20(tco2).balanceOf(address(this));
-            if (balance < 1e15) continue;
-            uint256 amountToRetire = remainingAmount > balance
-                ? balance
-                : remainingAmount;
-            IToucanCarbonOffsets(tco2).retire(amountToRetire);
-            remainingAmount -= amountToRetire;
+        while (i < _tco2s.length) {
+            if (_amounts[i] < 1e15) continue;
```
Comment (kargakis) [inline contracts/OffsetHelper.sol:311]: We can remove this check now.

Comment (lazaralex98) [reply contracts/OffsetHelper.sol:311]: Will do, but can you please remind me why is that? I remember you telling me this once before, but I can't remember the reason.

**Comment by kargakis** [comment]
There are also offset helper address at the end of the README file, can we remove them?

File: contracts/OffsetHelper.sol (hunk 1 above)
Comment (kargakis) [reply contracts/OffsetHelper.sol:311]: We removed the kilo limitation from the RetirementCertificates contract before releasing in the end.

**Review by kargakis — APPROVED** [review]

**Pull request merged**